concourse-resource-git.dockerfile

```diff
@@ -9,181 +11,165 @@ RUN apk --no-cache add \
+&& rm -rf /root/git-resource \
+&& cd /usr/libexec/git-core \
+&& rm -f \
+      git-add \
+      git-add--interactive \
+      git-annotate \
+      git-apply \
+      git-archimport \
+      git-archive \
+      git-bisect--helper \
+      git-blame \
+      git-branch \
+      git-bundle \
+      git-credential-cache \
+      git-credential-cache--daemon \
+      git-credential-store \
+      git-cat-file \
+      git-check-attr \
+      git-check-ignore \
+      git-check-mailmap \
+      git-check-ref-format \
+      git-checkout \
+      git-checkout-index \
+      git-cherry \
+      git-cherry-pick \
+      git-clean \
+      git-clone \
+      git-column \
+      git-commit \
+      git-commit-tree \
+      git-config \
+      git-count-objects \
+      git-credential \
+      git-cvsexportcommit \
+      git-cvsimport \
+      git-cvsserver \
+      git-describe \
+      git-diff \
+      git-diff-files \
+      git-diff-index \
+      git-diff-tree \
+      git-difftool \
+      git-fast-export \
+      git-fast-import \
+      git-fetch \
+      git-fetch-pack \
+      git-fmt-merge-msg \
+      git-for-each-ref \
+      git-format-patch \
+      git-fsck \
+      git-fsck-objects \
+      git-gc \
+      git-get-tar-commit-id \
+      git-grep \
+      git-hash-object \
+      git-help \
+      git-http-backend\
+      git-imap-send \
+      git-index-pack \
+      git-init \
+      git-init-db \
+      git-lfs \
+      git-log \
+      git-ls-files \
+      git-ls-remote \
+      git-ls-tree \
+      git-mailinfo \
+      git-mailsplit \
+      git-merge \
+      git-mktag \
+      git-mktree \
+      git-mv \
+      git-name-rev \
+      git-notes \
+      git-p4 \
+      git-pack-objects \
+      git-pack-redundant \
+      git-pack-refs \
+      git-patch-id \
+      git-peek-remote \
+      git-prune \
+      git-prune-packed \
+      git-push \
+      git-read-tree \
+      git-reflog \
+      git-relink \
+      git-remote \
+      git-remote-ext \
+      git-remote-fd \
+      git-remote-testsvn \
+      git-repack \
+      git-replace \
+      git-repo-config \
+      git-rerere \
+      git-reset \
+      git-rev-list \
+      git-rev-parse \
+      git-revert \
+      git-rm \
+      git-send-email \
+      git-send-pack \
+      git-shell \
+      git-shortlog \
+      git-show \
+      git-show-branch \
+      git-show-index \
+      git-show-ref \
+      git-stage \
+      git-show-ref \
+      git-stage \
+      git-status \
+      git-stripspace \
+      git-svn \
+      git-symbolic-ref \
+      git-tag \
+      git-tar-tree \
+      git-unpack-file \
+      git-unpack-objects \
+      git-update-index \
+      git-update-ref \
+      git-update-server-info \
+      git-upload-archive \
+      git-var \
+      git-verify-pack \
+      git-verify-tag \
+      git-whatchanged \
+      git-write-tree \
+&& ln -s /usr/libexec/git-core/git /usr/libexec/git-core/git-merge \
```

Comment:
nit: since `/usr/libexec/git-core` is already the working directory at this point, you could omit this part. However, keeping it might improve readability so I would let this up to you.

Reply:
generally-speaking, I agree. however, not all ln-implementation seem to be smart enough about relative paths, which motivated me to use absolute paths